server/models/post.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const PostSchema = mongoose.Schema(
+    {
+        userId: {
+            type: mongoose.Schema.Types.ObjectId, 
+            required: true,
+        },
+        description: {
+            type: String,
+            required: true,
+        },
+    },
+    {
+        timestamps: true,
```

Comment:
Not too familiar on this, is this how to add a timestamp field on the document? if it is, looks good